etna/ciim/constants.py

```diff
@@ -753,3 +754,5 @@ class Display(StrEnum):
         "long_display_name": "Paper catalogues available to view at The National Archives",
     },
 ]
+
+SEARCH_URL_RETAIN_DELTA = timezone.timedelta(hours=48)
```

Comment:
@TNA-Allan I don't feel like this app is the right place for this value. I would suggest just adding it to `etna/records/views/records.py` as a global

Reply:
This is moved to records.py